diff -uraN gnome-do-0.3.1.orig/Do/gnome-do.in gnome-do-0.3.1/Do/gnome-do.in
--- gnome-do-0.3.1.orig/Do/gnome-do.in	2008-02-29 20:41:52.000000000 +0100
+++ gnome-do-0.3.1/Do/gnome-do.in	2008-02-29 20:41:17.000000000 +0100
@@ -6,12 +6,12 @@
 TOMBOY_LIB=`pkg-config --variable=libdir tomboy-addins`/tomboy
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$TOMBOY_LIB
 
-if [ "x$SCRIPT_PATH" = "x@abs_srcdir@/bin/Debug" ] ||
-   [ "x$SCRIPT_PATH" = "x@abs_srcdir@" ] ; then
+if [ "$SCRIPT_PATH" = "@libdir@/bin/Debug" ] ||
+   [ "$SCRIPT_PATH" = "@libdir@" ] ; then
     echo "*** Running uninstalled ***"
-    GNOME_DO_EXE="@abs_srcdir@/bin/Debug/Do.exe"
+    GNOME_DO_EXE="@libdir@/bin/Debug/Do.exe"
 else
-    GNOME_DO_EXE="@prefix@/lib/@PACKAGE@/Do.exe"
+    GNOME_DO_EXE="@libdir@/@PACKAGE@/Do.exe"
 fi
 
 exec mono "$GNOME_DO_EXE" "$@"
diff -uraN gnome-do-0.3.1.orig/Do/Makefile.in gnome-do-0.3.1/Do/Makefile.in
--- gnome-do-0.3.1.orig/Do/Makefile.in	2008-02-29 20:41:52.000000000 +0100
+++ gnome-do-0.3.1/Do/Makefile.in	2008-02-29 20:42:12.000000000 +0100
@@ -337,9 +337,9 @@
 bin_SCRIPTS = $(BINARIES)
 linuxdesktopapplicationsdir = @prefix@/share/applications
 linuxdesktopapplications_DATA = $(LINUX_DESKTOPAPPLICATIONS)
-programfilesdir = @prefix@/lib/@PACKAGE@
+programfilesdir = @libdir@/@PACKAGE@
 programfiles_DATA = $(PROGRAMFILES)
-linuxpkgconfigdir = @prefix@/lib/pkgconfig
+linuxpkgconfigdir = @libdir@/pkgconfig
 linuxpkgconfig_DATA = $(LINUX_PKGCONFIG)
 all: all-am
 
diff -uraN gnome-do-0.3.1.orig/Do.Addins/do.addins.pc.in gnome-do-0.3.1/Do.Addins/do.addins.pc.in
--- gnome-do-0.3.1.orig/Do.Addins/do.addins.pc.in	2008-02-29 20:41:52.000000000 +0100
+++ gnome-do-0.3.1/Do.Addins/do.addins.pc.in	2008-02-29 20:42:12.000000000 +0100
@@ -3,4 +3,4 @@
 Version: 0.1
 
 Requires: 
-Libs: -r:@prefix@/lib/@PACKAGE@/Do.Addins.dll
+Libs: -r:@libdir@/@PACKAGE@/Do.Addins.dll
diff -uraN gnome-do-0.3.1.orig/Do.Addins/Makefile.in gnome-do-0.3.1/Do.Addins/Makefile.in
--- gnome-do-0.3.1.orig/Do.Addins/Makefile.in	2008-02-29 20:41:52.000000000 +0100
+++ gnome-do-0.3.1/Do.Addins/Makefile.in	2008-02-29 20:42:12.000000000 +0100
@@ -299,9 +299,9 @@
 bin_SCRIPTS = $(BINARIES)
 linuxdesktopapplicationsdir = @prefix@/share/applications
 linuxdesktopapplications_DATA = $(LINUX_DESKTOPAPPLICATIONS)
-programfilesdir = @prefix@/lib/@PACKAGE@
+programfilesdir = @libdir@/@PACKAGE@
 programfiles_DATA = $(PROGRAMFILES)
-linuxpkgconfigdir = @prefix@/lib/pkgconfig
+linuxpkgconfigdir = @libdir@/pkgconfig
 linuxpkgconfig_DATA = $(LINUX_PKGCONFIG)
 all: all-am
 
diff -uraN gnome-do-0.3.1.orig/Do.DBus/do.dbus.pc.in gnome-do-0.3.1/Do.DBus/do.dbus.pc.in
--- gnome-do-0.3.1.orig/Do.DBus/do.dbus.pc.in	2008-02-29 20:41:52.000000000 +0100
+++ gnome-do-0.3.1/Do.DBus/do.dbus.pc.in	2008-02-29 20:42:12.000000000 +0100
@@ -3,4 +3,4 @@
 Version: 0.1
 
 Requires: 
-Libs: -r:@prefix@/lib/@PACKAGE@/Do.DBus.dll
+Libs: -r:@libdir@/@PACKAGE@/Do.DBus.dll
diff -uraN gnome-do-0.3.1.orig/Do.DBus/Makefile.in gnome-do-0.3.1/Do.DBus/Makefile.in
--- gnome-do-0.3.1.orig/Do.DBus/Makefile.in	2008-02-29 20:41:52.000000000 +0100
+++ gnome-do-0.3.1/Do.DBus/Makefile.in	2008-02-29 20:42:12.000000000 +0100
@@ -269,9 +269,9 @@
 bin_SCRIPTS = $(BINARIES)
 linuxdesktopapplicationsdir = @prefix@/share/applications
 linuxdesktopapplications_DATA = $(LINUX_DESKTOPAPPLICATIONS)
-programfilesdir = @prefix@/lib/@PACKAGE@
+programfilesdir = @libdir@/@PACKAGE@
 programfiles_DATA = $(PROGRAMFILES)
-linuxpkgconfigdir = @prefix@/lib/pkgconfig
+linuxpkgconfigdir = @libdir@/pkgconfig
 linuxpkgconfig_DATA = $(LINUX_PKGCONFIG)
 all: all-am
 
diff -uraN gnome-do-0.3.1.orig/Makefile.include gnome-do-0.3.1/Makefile.include
--- gnome-do-0.3.1.orig/Makefile.include	2008-02-29 20:41:52.000000000 +0100
+++ gnome-do-0.3.1/Makefile.include	2008-02-29 20:42:12.000000000 +0100
@@ -31,7 +31,7 @@
 
 linuxdesktopapplicationsdir = @prefix@/share/applications
 linuxdesktopapplications_DATA = $(LINUX_DESKTOPAPPLICATIONS)
-programfilesdir = @prefix@/lib/@PACKAGE@
+programfilesdir = @libdir@/@PACKAGE@
 programfiles_DATA = $(PROGRAMFILES)
-linuxpkgconfigdir = @prefix@/lib/pkgconfig
+linuxpkgconfigdir = @libdir@/pkgconfig
 linuxpkgconfig_DATA = $(LINUX_PKGCONFIG)
