--- gnome-do-0.5.0.1/Do/gnome-do.in.launcher	2008-06-09 20:48:11.000000000 -0400
+++ gnome-do-0.5.0.1/Do/gnome-do.in	2008-07-06 21:29:51.000000000 -0400
@@ -1,15 +1,6 @@
 #!/bin/sh
 
-SCRIPT_PATH=`dirname $0`
-SCRIPT_PATH=`cd $SCRIPT_PATH; pwd`
-
-if [ "x$SCRIPT_PATH" = "x@abs_srcdir@/bin/Debug" ] ||
-   [ "x$SCRIPT_PATH" = "x@abs_srcdir@" ] ; then
-    echo "*** Running uninstalled ***"
-    GNOME_DO_EXE="@abs_srcdir@/bin/Debug/Do.exe"
-else
-    GNOME_DO_EXE="@expanded_libdir@/@PACKAGE@/Do.exe"
-fi
+GNOME_DO_EXE="@expanded_libdir@/@PACKAGE@/Do.exe"
 
 # Send the dbus Summon signal right away for fastest response if
 # Do is already running.
