diff -uNr gnome-do-0.8.3.1.old/libdo/Makefile.am gnome-do-0.8.3.1/libdo/Makefile.am
--- gnome-do-0.8.3.1.old/libdo/Makefile.am	2009-06-29 02:19:29.000000000 +0200
+++ gnome-do-0.8.3.1/libdo/Makefile.am	2010-10-29 00:17:40.000000000 +0200
@@ -3,7 +3,6 @@
 AM_CFLAGS = -DG_LOG_DOMAIN=\"libdo\"	\
 	    -DEGG_COMPILATION 		\
 	    -DGTK_DISABLE_DEPRECATED 	\
-	    -DGDK_DISABLE_DEPRECATED 	\
 	    -DG_DISABLE_DEPRECATED	\
 	    $(LIBDO_CFLAGS)
 
diff -uNr gnome-do-0.8.3.1.old/libdo/Makefile.in gnome-do-0.8.3.1/libdo/Makefile.in
--- gnome-do-0.8.3.1.old/libdo/Makefile.in	2009-12-09 01:17:17.000000000 +0100
+++ gnome-do-0.8.3.1/libdo/Makefile.in	2010-10-29 00:17:34.000000000 +0200
@@ -281,7 +281,6 @@
 AM_CFLAGS = -DG_LOG_DOMAIN=\"libdo\"	\
 	    -DEGG_COMPILATION 		\
 	    -DGTK_DISABLE_DEPRECATED 	\
-	    -DGDK_DISABLE_DEPRECATED 	\
 	    -DG_DISABLE_DEPRECATED	\
 	    $(LIBDO_CFLAGS)
 
