diff -uraN do-0.3.orig/Do/gnome-do.in do-0.3/Do/gnome-do.in
--- do-0.3.orig/Do/gnome-do.in	2008-01-22 01:49:45.000000000 +0100
+++ do-0.3/Do/gnome-do.in	2008-01-22 02:03:06.000000000 +0100
@@ -11,7 +11,7 @@
     echo "*** Running uninstalled ***"
     GNOME_DO_EXE="@abs_srcdir@/bin/Debug/Do.exe"
 else
-    GNOME_DO_EXE="@prefix@/lib/@PACKAGE@/Do.exe"
+    GNOME_DO_EXE="@libdir@/@PACKAGE@/Do.exe"
 fi
 
 exec mono "$GNOME_DO_EXE" "$@"
diff -uraN do-0.3.orig/Do/Makefile.in do-0.3/Do/Makefile.in
--- do-0.3.orig/Do/Makefile.in	2008-01-22 01:49:45.000000000 +0100
+++ do-0.3/Do/Makefile.in	2008-01-22 02:03:34.000000000 +0100
@@ -345,9 +345,9 @@
 bin_SCRIPTS = $(BINARIES)
 linuxdesktopapplicationsdir = @prefix@/share/applications
 linuxdesktopapplications_DATA = $(LINUX_DESKTOPAPPLICATIONS)
-programfilesdir = @prefix@/lib/@PACKAGE@
+programfilesdir = @libdir@/@PACKAGE@
 programfiles_DATA = $(PROGRAMFILES)
-linuxpkgconfigdir = @prefix@/lib/pkgconfig
+linuxpkgconfigdir = @libdir@/pkgconfig
 linuxpkgconfig_DATA = $(LINUX_PKGCONFIG)
 all: all-am
 
diff -uraN do-0.3.orig/Do.Addins/do.addins.pc.in do-0.3/Do.Addins/do.addins.pc.in
--- do-0.3.orig/Do.Addins/do.addins.pc.in	2008-01-22 01:49:45.000000000 +0100
+++ do-0.3/Do.Addins/do.addins.pc.in	2008-01-22 02:02:09.000000000 +0100
@@ -3,4 +3,4 @@
 Version: 0.1
 
 Requires: 
-Libs: -r:@prefix@/lib/@PACKAGE@/Do.Addins.dll
+Libs: -r:@libdir@/@PACKAGE@/Do.Addins.dll
diff -uraN do-0.3.orig/Do.Addins/Makefile.in do-0.3/Do.Addins/Makefile.in
--- do-0.3.orig/Do.Addins/Makefile.in	2008-01-22 01:49:45.000000000 +0100
+++ do-0.3/Do.Addins/Makefile.in	2008-01-22 02:02:46.000000000 +0100
@@ -307,9 +307,9 @@
 bin_SCRIPTS = $(BINARIES)
 linuxdesktopapplicationsdir = @prefix@/share/applications
 linuxdesktopapplications_DATA = $(LINUX_DESKTOPAPPLICATIONS)
-programfilesdir = @prefix@/lib/@PACKAGE@
+programfilesdir = @libdir@/@PACKAGE@
 programfiles_DATA = $(PROGRAMFILES)
-linuxpkgconfigdir = @prefix@/lib/pkgconfig
+linuxpkgconfigdir = @libdir@/pkgconfig
 linuxpkgconfig_DATA = $(LINUX_PKGCONFIG)
 all: all-am
 
diff -uraN do-0.3.orig/Do.DBus/do.dbus.pc.in do-0.3/Do.DBus/do.dbus.pc.in
--- do-0.3.orig/Do.DBus/do.dbus.pc.in	2008-01-22 01:49:45.000000000 +0100
+++ do-0.3/Do.DBus/do.dbus.pc.in	2008-01-22 02:00:28.000000000 +0100
@@ -3,4 +3,4 @@
 Version: 0.1
 
 Requires: 
-Libs: -r:@prefix@/lib/@PACKAGE@/Do.DBus.dll
+Libs: -r:@libdir@/@PACKAGE@/Do.DBus.dll
diff -uraN do-0.3.orig/Do.DBus/Makefile.in do-0.3/Do.DBus/Makefile.in
--- do-0.3.orig/Do.DBus/Makefile.in	2008-01-22 01:49:45.000000000 +0100
+++ do-0.3/Do.DBus/Makefile.in	2008-01-22 02:01:15.000000000 +0100
@@ -278,9 +278,9 @@
 bin_SCRIPTS = $(BINARIES)
 linuxdesktopapplicationsdir = @prefix@/share/applications
 linuxdesktopapplications_DATA = $(LINUX_DESKTOPAPPLICATIONS)
-programfilesdir = @prefix@/lib/@PACKAGE@
+programfilesdir = @libdir@/@PACKAGE@
 programfiles_DATA = $(PROGRAMFILES)
-linuxpkgconfigdir = @prefix@/lib/pkgconfig
+linuxpkgconfigdir = @libdir@/pkgconfig
 linuxpkgconfig_DATA = $(LINUX_PKGCONFIG)
 all: all-am
 
diff -uraN do-0.3.orig/Makefile.include do-0.3/Makefile.include
--- do-0.3.orig/Makefile.include	2008-01-22 01:49:45.000000000 +0100
+++ do-0.3/Makefile.include	2008-01-22 02:01:53.000000000 +0100
@@ -31,7 +31,7 @@
 
 linuxdesktopapplicationsdir = @prefix@/share/applications
 linuxdesktopapplications_DATA = $(LINUX_DESKTOPAPPLICATIONS)
-programfilesdir = @prefix@/lib/@PACKAGE@
+programfilesdir = @libdir@/@PACKAGE@
 programfiles_DATA = $(PROGRAMFILES)
-linuxpkgconfigdir = @prefix@/lib/pkgconfig
+linuxpkgconfigdir = @libdir@/pkgconfig
 linuxpkgconfig_DATA = $(LINUX_PKGCONFIG)
