diff -uraN gnome-do-0.4.0.1.orig/Do/Do.exe.config.in gnome-do-0.4.0.1/Do/Do.exe.config.in
--- gnome-do-0.4.0.1.orig/Do/Do.exe.config.in	2008-03-29 12:15:13.033559861 +0100
+++ gnome-do-0.4.0.1/Do/Do.exe.config.in	2008-03-29 12:19:04.558800507 +0100
@@ -1,4 +1,4 @@
 <configuration>
 	<dllmap dll="libc" target="libc.so.6"/>
-	<dllmap dll="libdo" target="@prefix@/lib/@PACKAGE@/libdo.so"/>
+	<dllmap dll="libdo" target="@libdir@/@PACKAGE@/libdo.so"/>
 </configuration>
diff -uraN gnome-do-0.4.0.1.orig/Do/gnome-do.in gnome-do-0.4.0.1/Do/gnome-do.in
--- gnome-do-0.4.0.1.orig/Do/gnome-do.in	2008-03-29 12:15:13.043566443 +0100
+++ gnome-do-0.4.0.1/Do/gnome-do.in	2008-03-29 12:18:38.478801211 +0100
@@ -8,7 +8,7 @@
     echo "*** Running uninstalled ***"
     GNOME_DO_EXE="@abs_srcdir@/bin/Debug/Do.exe"
 else
-    GNOME_DO_EXE="@prefix@/lib/@PACKAGE@/Do.exe"
+    GNOME_DO_EXE="@libdir@/@PACKAGE@/Do.exe"
 fi
 
 # Send the dbus Summon signal right away for fastest response if
diff -uraN gnome-do-0.4.0.1.orig/Do/Makefile.in gnome-do-0.4.0.1/Do/Makefile.in
--- gnome-do-0.4.0.1.orig/Do/Makefile.in	2008-03-29 12:15:13.044550367 +0100
+++ gnome-do-0.4.0.1/Do/Makefile.in	2008-03-29 12:19:40.007550124 +0100
@@ -363,9 +363,9 @@
 bin_SCRIPTS = $(BINARIES)
 linuxdesktopapplicationsdir = @prefix@/share/applications
 linuxdesktopapplications_DATA = $(LINUX_DESKTOPAPPLICATIONS)
-programfilesdir = @prefix@/lib/@PACKAGE@
+programfilesdir = @libdir@/@PACKAGE@
 programfiles_DATA = $(PROGRAMFILES)
-linuxpkgconfigdir = @prefix@/lib/pkgconfig
+linuxpkgconfigdir = @libdir@/pkgconfig
 linuxpkgconfig_DATA = $(LINUX_PKGCONFIG)
 all: all-am
 
diff -uraN gnome-do-0.4.0.1.orig/Do.Addins/do.addins.pc.in gnome-do-0.4.0.1/Do.Addins/do.addins.pc.in
--- gnome-do-0.4.0.1.orig/Do.Addins/do.addins.pc.in	2008-03-29 12:15:13.030550260 +0100
+++ gnome-do-0.4.0.1/Do.Addins/do.addins.pc.in	2008-03-29 12:18:15.910803443 +0100
@@ -3,4 +3,4 @@
 Version: 0.1
 
 Requires: 
-Libs: -r:@prefix@/lib/@PACKAGE@/Do.Addins.dll
+Libs: -r:@libdir@/@PACKAGE@/Do.Addins.dll
diff -uraN gnome-do-0.4.0.1.orig/Do.Addins/Makefile.in gnome-do-0.4.0.1/Do.Addins/Makefile.in
--- gnome-do-0.4.0.1.orig/Do.Addins/Makefile.in	2008-03-29 12:15:13.030550260 +0100
+++ gnome-do-0.4.0.1/Do.Addins/Makefile.in	2008-03-29 12:17:52.423800629 +0100
@@ -349,9 +349,9 @@
 bin_SCRIPTS = $(BINARIES)
 linuxdesktopapplicationsdir = @prefix@/share/applications
 linuxdesktopapplications_DATA = $(LINUX_DESKTOPAPPLICATIONS)
-programfilesdir = @prefix@/lib/@PACKAGE@
+programfilesdir = @libdir@/@PACKAGE@
 programfiles_DATA = $(PROGRAMFILES)
-linuxpkgconfigdir = @prefix@/lib/pkgconfig
+linuxpkgconfigdir = @libdir@/pkgconfig
 linuxpkgconfig_DATA = $(LINUX_PKGCONFIG)
 all: all-am
 
diff -uraN gnome-do-0.4.0.1.orig/Do.DBus/do.dbus.pc.in gnome-do-0.4.0.1/Do.DBus/do.dbus.pc.in
--- gnome-do-0.4.0.1.orig/Do.DBus/do.dbus.pc.in	2008-03-29 12:15:13.007550124 +0100
+++ gnome-do-0.4.0.1/Do.DBus/do.dbus.pc.in	2008-03-29 12:15:39.758550251 +0100
@@ -3,4 +3,4 @@
 Version: 0.1
 
 Requires: 
-Libs: -r:@prefix@/lib/@PACKAGE@/Do.DBus.dll
+Libs: -r:@libdir@/@PACKAGE@/Do.DBus.dll
diff -uraN gnome-do-0.4.0.1.orig/Do.DBus/Makefile.in gnome-do-0.4.0.1/Do.DBus/Makefile.in
--- gnome-do-0.4.0.1.orig/Do.DBus/Makefile.in	2008-03-29 12:15:13.007550124 +0100
+++ gnome-do-0.4.0.1/Do.DBus/Makefile.in	2008-03-29 12:16:22.315801398 +0100
@@ -298,9 +298,9 @@
 bin_SCRIPTS = $(BINARIES)
 linuxdesktopapplicationsdir = @prefix@/share/applications
 linuxdesktopapplications_DATA = $(LINUX_DESKTOPAPPLICATIONS)
-programfilesdir = @prefix@/lib/@PACKAGE@
+programfilesdir = @libdir@/@PACKAGE@
 programfiles_DATA = $(PROGRAMFILES)
-linuxpkgconfigdir = @prefix@/lib/pkgconfig
+linuxpkgconfigdir = @libdir@/pkgconfig
 linuxpkgconfig_DATA = $(LINUX_PKGCONFIG)
 all: all-am
 
diff -uraN gnome-do-0.4.0.1.orig/Makefile.include gnome-do-0.4.0.1/Makefile.include
--- gnome-do-0.4.0.1.orig/Makefile.include	2008-03-29 12:15:13.008553604 +0100
+++ gnome-do-0.4.0.1/Makefile.include	2008-03-29 12:17:17.014803841 +0100
@@ -31,7 +31,7 @@
 
 linuxdesktopapplicationsdir = @prefix@/share/applications
 linuxdesktopapplications_DATA = $(LINUX_DESKTOPAPPLICATIONS)
-programfilesdir = @prefix@/lib/@PACKAGE@
+programfilesdir = @libdir@/@PACKAGE@
 programfiles_DATA = $(PROGRAMFILES)
-linuxpkgconfigdir = @prefix@/lib/pkgconfig
+linuxpkgconfigdir = @libdir@/pkgconfig
 linuxpkgconfig_DATA = $(LINUX_PKGCONFIG)
